incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -206,19 +206,19 @@ However, if you actually try this, you'll get an error:
 ```
 $ kubectl delete namespace parent
 # Output:
-Error from server (Forbidden): admission webhook "namespaces.hnc.x-k8s.io" denied the request: Please set allowCascadingDelete first either in the parent namespace or in all the subnamespaces.
- Subnamespace(s) without allowCascadingDelete set: [child].
+Error from server (Forbidden): admission webhook "namespaces.hnc.x-k8s.io" denied the request: Please set allowCascadingDeletion first either in the parent namespace or in all the subnamespaces.
+ Subnamespace(s) without allowCascadingDeletion set: [child].
 ```
 
 These errors are there for your protection. Deleting namespaces is very
 dangerous, and deleting _subnamespaces_ can result in entire subtrees of
 namespaces being deleted as well. Therefore, if deleting a namespace (or
 subnamespace) would result in the deletion of any namespace _other than the one
 explicitly being deleted_, HNC requires that you must specify the
-`allowCascadingDelete` field on either all the namespaces that will be
+`allowCascadingDeletion` field on either all the namespaces that will be
 implicitly deleted, or any of their ancestors.
 
-The `allowCascadingDelete` field is a bit like `rm -rf` in a Linux shell.
+The `allowCascadingDeletion` field is a bit like `rm -rf` in a Linux shell.
```

Comment:
Can you please add a note that in v0.5.x, HNC uses v1a1 and this field is called `allowCascadingDelete`?

Reply:
Done. Replaced the v0.4.x warning.